### ISSUE TITLE
Make NGP buttons obvious as to what they do

### ISSUE THREAD

**Issue by warpdragon** [OpenAngelArena/oaa#249]
Maybe like 
![image](https://cloud.githubusercontent.com/assets/13878439/24157551/0a598ad6-0e30-11e7-9b9f-ad46aeb28455.png)

**Comment by ProstoSanja** [comment]
I'll deal with this today, we will see how it works

**Comment by chrisinajar** [comment]
they look like this now
![image](https://cloud.githubusercontent.com/assets/422331/24157773/94ad77c4-0e17-11e7-8a91-fd043b3aef37.png)

lets wait on art guys before doing a full revamp. it's unlikely a redo will look substantially better, and the current button layout is designed to be a reference to WoW

**Comment by Haganeko** [comment]
"Builds into" is a bit silly since we decided on simple upgrade cores - they build into nearly everything.

**Comment by chrisinajar** [comment]
For core 1 sure, but 2 or 3 would be nice to mark things like which tier you can get a lucience with. Also, i believe we plan on removing combiners in favor of just upgrade cores, so knowing which tier makes things like charge bkb would also be nice.

really all that would go in that space otherwise would be more descriptive text, since the mouseover effects work on the builds into items (so you can see stats and stuff) i still really like it.

**Comment by ProstoSanja** [comment]
Maybe we should make it like warpdragon proposed, and add mouseover for ngp buttons? And remain text above them, and remove combines into. 
(That English though)

**Comment by Haganeko** [comment]
Problem is there will be some hundred items to display on "builds into", especially for the higher cores, which can substitute for lower ones.

**Comment by ProstoSanja** [comment]
So, making like warpdragon drew above?

**Comment by warpdragon** [comment]
I like the captions next to the buttons now. That's awesome. Also I completely agree with removing combiners. They're functionally indistinct to upgrade cores anyway and they're just another item to clutter your inventory.

The problem with "builds into" is you'll never fit them all in. Highlight the "coolest" ones maybe?

@ProstoSanja chris is content with the current layout atleast there's captions saying what the buttons do. it's lower on priority list than the current project (bosses and reflex items) so just leave it for now?

**Comment by ProstoSanja** [comment]
I suggested it, because it is a 30 min thing, and I will anyway deal with it today, as Chris asked to fix a text line there, so I might do this all together.

**Comment by Haganeko** [comment]
Could we make the ngp table interact with the shop interface to display some "builds into" information? Maybe a button?
"Builds into" will be primarily useful for unique boss items,  anyways.

**Comment by Chronophylos** [comment]
I don't think this is possibly, but we wanted to make our own custom shop anyway.

**Comment by chrisinajar** [comment]
define `interact`, valve does give a lot of API's for shop stuff. I know we can at least add them to the quick buy slot, i *think* you can do a Show In Show gesture as well, not sure about the later.

**Comment by chrisinajar** [comment]
unique boss items will build into exactly 1 item, i actually think they're less useful there.

**Comment by Haganeko** [comment]
Clarifying: We could make the dropped items clickable to show them in the shop, with their build tree. The space in the ngp prompt is not enough to show what an upgrade core can build into (I'm unsure even the shop has the space), and as Chris said "builds into" for unique boss items is mostly to show the stats of the item it makes.

**Comment by Chronophylos** [comment]
@ProstoSanja we need a custom shop for better interaction with our Gold API as well.

**Comment by ProstoSanja** [comment]
Working with this at #255

**Comment by Haganeko** [comment]
Close this?

**Comment by Chronophylos** [comment]
Technically #255 is working on it. So not now.
